README.md

```diff
@@ -441,6 +442,36 @@ The inline attributes decides whether the input fields and selected tags will be
 
 ![img](docs/inline-false.png)
 
+_This attribute is deprecated and will be removed in v7.x.x, see [inputFieldPosition](#inputFieldPosition)._
+
+<a name="inputFieldPosition"></a>
+##### inputFieldPosition (optional, defaults to `inline`)
+The inputFieldPosition attribute decides the positioning of the input field relative to the tags. Can be one of `inline`, `top` or `bottom`.
+
+```
+<ReactTags
+    inputFieldPosition="inline"
+    ...>
+```
+
+![img](docs/input-field-position-inline.png)
+
+```
+<ReactTags
+    inputFieldPosition="top"
+    ...>
+```
+
+![img](docs/input-field-position-top.png)
```

Comment:
@atiqueansari1987 there is no file named `input-field-position-top`. Seems like a typo, can you rename `iinput-field-position-bottom.png` to `input-field-position-top`.

Reply:
Done